src/renderToStream.ts

```diff
@@ -191,6 +193,7 @@ async function renderToWebStream(
   if (didError) throw firstErr
   if (disable) await allReady
   if (didError) throw firstErr
+  const { createReadableWrapper } = await import('./renderToStream/createReadableWrapper')
```

Comment:
I don't think we need this change?

Reply:
I wasn't sure. I'm testing it within Deno now